src/main/java/org/folio/innreach/domain/service/impl/BatchDomainEventProcessor.java

```diff
@@ -36,7 +39,12 @@ public <T> void process(List<DomainEvent<T>> batch, Consumer<DomainEvent<T>> rec
       try {
         executionService.runTenantScoped(tenantId,
           () -> processTenantEvents(events, recordProcessor));
-      } catch (ListenerExecutionFailedException | FeignException ex) {
+      }
+      catch (ServiceSuspendedException | FeignException | InnReachConnectionException | SocketTimeOutExceptionWrapper e) {
+        log.info("exception thrown from process");
```

Comment:
Would be a good idea to log the actual exception thrown here (`log.info("Exception thrown from process", e)`) to make debugging in prod easier.